app/routes/holds.js

```diff
@@ -0,0 +1,49 @@
+const schema = require('./schemas/hold')
+const { get, post } = require('../api')
+
+module.exports = [{
+  method: 'GET',
+  path: '/payment-holds',
+  options: {
+    handler: async (request, h) => {
+      const paymentHoldsResponse = await get('/payment-holds')
+      return h.view('payment-holds', { paymentHolds: paymentHoldsResponse.payload.paymentHolds })
+    }
+  }
+}, {
+  method: 'GET',
+  path: '/add-payment-hold',
+  options: {
+    handler: async (request, h) => {
+      const paymentHoldCategoriesResponse = await get('/payment-hold-categories')
+      return h.view('add-payment-hold', { paymentHoldCategories: paymentHoldCategoriesResponse.payload.paymentHoldCategories })
+    }
+  }
+},
+{
+  method: 'POST',
+  path: '/add-payment-hold',
+  options: {
+    validate: {
+      payload: schema,
+      failAction: async (request, h, error) => {
+        const paymentHoldCategoriesResponse = await get('/payment-hold-categories')
+        return h.view('add-payment-hold', { paymentHoldCategories: paymentHoldCategoriesResponse.payload.paymentHoldCategories, errors: error }).code(400).takeover()
+      }
+    },
+    handler: async (request, h) => {
+      await post('/add-payment-hold', { holdCategoryId: request.payload.holdCategoryId, frn: request.payload.frn }, null)
+      return h.redirect('/payment-holds')
+    }
+  }
+},
+{
+  method: 'POST',
```

Comment:
Should this be `PUT`? Doesn't feel right having two `POST`

Reply:
Can't do PUT from HTML.